spec/support/helpers/features.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module SpecHelpers
+  module Features
+    attr_reader :current_user
+
+    def assume_logged_in(user = nil, admin: false)
+      user ||= user || admin ? create(:user, :admin) : create(:user)
```

Comment:
This logic seems to me a bit complicated and not useful. What do you think about changing it to this?
```suggestion
      user ||= admin.present? ? create(:user, :admin) : create(:user)
```
or even this:
```suggestion
      user ||= admin ? create(:user, :admin) : create(:user)
```

Reply:
I'd choose the second variant.

Reply:
I'd choose the second variant.

Reply:
I'd choose the second variant.

Reply:
I'd choose the second variant.

Reply:
i think it's complicated because of naming. User can be a local variable and passed variable.
```ruby
def assume_logged_in(user = nil, admin: false)
  local_user ||= user || admin ? create(:user, :admin) : create(:user)
   @current_user = local_user

   login_as(local_user)
end
```

Reply:
Yes, this is clearer 👍